vulnerabilities/importers/ruby.py

```diff
@@ -7,132 +7,196 @@
 # See https://aboutcode.org for more information about nexB OSS projects.
 #
 
-import asyncio
+import logging
+from typing import Iterable
 from typing import List
-from typing import Set
 
 from dateutil.parser import parse
 from packageurl import PackageURL
 from pytz import UTC
-from univers.version_range import VersionRange
-from univers.versions import SemverVersion
+from univers.version_range import GemVersionRange
+from univers.versions import RubygemsVersion
 
 from vulnerabilities.importer import AdvisoryData
+from vulnerabilities.importer import AffectedPackage
+from vulnerabilities.importer import GitConfig
 from vulnerabilities.importer import GitImporter
 from vulnerabilities.importer import Reference
+from vulnerabilities.importer import VulnerabilitySeverity
 from vulnerabilities.package_managers import RubyVersionAPI
+from vulnerabilities.severity_systems import SCORING_SYSTEMS
+from vulnerabilities.utils import build_description
 from vulnerabilities.utils import load_yaml
-from vulnerabilities.utils import nearest_patched_package
+
+logger = logging.getLogger(__name__)
 
 
 class RubyImporter(GitImporter):
-    def __enter__(self):
-        super(RubyImporter, self).__enter__()
+    license_url = "https://github.com/rubysec/ruby-advisory-db/blob/master/LICENSE.txt"
+    spdx_license_expression = "unknow"
+    config = GitConfig(
+        repository_url="https://github.com/rubysec/ruby-advisory-db.git",
+        working_directory="ruby",
+        branch="master",
+    )
+    cutoff_timestamp = 1
+
+    def __init__(self):
+        super().__init__(config=self.config, cutoff_timestamp=self.cutoff_timestamp)
```

Comment:
Version API needs to be handled at improver side, please check github and gitlab importer for same